src/weatherSource/WeatherSource.js

```diff
@@ -0,0 +1,129 @@
+import {
+    relative
+} from "path";
+
+const API_KEY = '953349aaa2569f9cd4821f8c2ffda23a'
+
+class WeatherAPI {
+
+    static _throw(text) {
+        throw text;
+    }
+
+    static _verify(params) {
+        let constructUrl;
+
+        if (params) {
+            params.latitude && params.longitude ? constructUrl = `lat=${params.latitude}&lon=${params.longitude}` :
```

Comment:
Hmmm. Mam tu lekkie wątpliwości - czy metoda this._throw jest na pewno potrzebna? Czy na końcu tego wszystkiego nie przydałby się średnik?
No i osobiście mam poczucie, że jak masz dłuższą konstrukcję if-else if-else to skrócona notacja a ? b : c raczej zmniejsza zamiast zwiększać czytelność. Moja sugestia byłaby, żeby jednak pisać pełną składnię blokową w takich sytuacjach - no, ale to takie moje wrażenia, nie traktuj tego jako jakiejś wiążącej prawdy objawionej.

Reply:
Metoda `_throw` była stworzona po to, aby wiadomo były rzucane wyjatki w przypadku źle podanych parametrów. Zależało mi na tym, aby w operatorze warunkowym w przypadku wartości `false`  rzucany był wyjątek. Jednak JS nie pozwala(przynajmniej na razie), aby to zrobić w taki sposób. 

Sam miałem wątpliwości co do struktury tego if else i operatorów warunkowych. Z początku myślałem nad samymi operatorami warunkowymi (bardziej mi się podobały warunki w jednej linijce) . Jednak to zostawiłem głosu większości.